PROGRAMS.md

```diff
@@ -114,7 +114,7 @@
 	<tr align="center">
 		<td>Pomodoro</td>
 		<td><code>pomodoro</code></td>
-		<td>02:40:00</td>
+		<td>02:10:00</td>
```

Comment:
Shouldn't it be `2:15:00` (4 * (25 + 5) + 15 = 135 minutes)?

Reply:
`Work` + `Short-Break` + `Work` + `Short-Break` + `Work` + `Short-Break` + `Work` + `Long-Break`

Reply:
Oh, you are right.